RecoLuminosity/LumiDB/scripts/pileupCalc.py

```diff
@@ -6,8 +6,8 @@
 import argparse
 from RecoLuminosity.LumiDB import pileupParser
 from RecoLuminosity.LumiDB import selectionParser
-from math import exp
-from math import sqrt
+import numpy as np
```

Comment:
This is a nice PR. Could the script use python3 instead? scipy is likely to be removed from the small stack of remaining python2 tools in the next days. I think it just needs the ```/usr/bin/env python``` to be changed to ```/usr/bin/env python3``` in the first line.

Reply:
Thanks for the suggestion, I will test it in python3 the coming days